src/game/Entities/CreatureLinkingMgr.h

```diff
@@ -71,6 +71,7 @@ enum CreatureLinkingFlags
 
     // Dynamic behaviour, out of combat
     FLAG_FOLLOW                     = 0x0200,
+    FLAG_FOLLOW_WITH_ANGLE			= 0x0201,
```

Comment:
FLAG_FOLLOW_WITH_ANGLE = 0x0201 is wrong enum mask in this case, because 0x0201 is a concatenation of FLAG_AGGRO_ON_AGGRO( 0x0001 ) | FLAG_FOLLOW( 0x0200 )

You should use FLAG_FOLLOW_WITH_ANGLE = 0x0400